docs/Lifecycle/ChangeControlBoard.md

```diff
@@ -0,0 +1,71 @@
+# **Change Control Board (CCB).**
+# Roles and responsibilities.
+
+CCB is a committee to which following responsibilities are delegated by stakeholders of SDL:
+1. Review and approve proposals made in scope of SDL Evolution
+2. Confirm delivery for readiness
+
+## **Roles**
+
+CCB include following roles:
+- Business Analyst (BA)
+- Architect
+- Test Lead Engineer
+- Developer Lead Engineer
+- CCB Group Manager
+
+### Business Analyst
+
+Following are responsibilities of BA:
+1. Proposal review
+	* Provide approval on proposal specification readiness
+	* Create set of requirements issues in respective repositories
+	* Create set of technical tasks in respective repositories
+	* Maintain links between proposals, requirements and technical tasks
+
+2. Delivery acceptance (overall review and acceptance)
```

Comment:
Acceptance of what? SDL delivery?
I'm not sure that it's a part of RM responsibilities.